### PR TITLE
Fix: Remove error on field when user starts typing or makes a change.

### PR DESCRIPTION
**Pull request by JonaszJestem**
created additional issue to fix those warnings:
https://github.com/SoftwareBrothers/admin-bro/issues/714

### CODE REVIEW

File: src/frontend/hooks/use-record/update-record.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const updateRecord = (
     populatedModified = true
   }
 
+  delete previousRecord.errors[property]
```
Comment (JonaszJestem) [inline src/frontend/hooks/use-record/update-record.ts:49]: Actual fix

File: src/frontend/components/property-type/default-type/edit.tsx [hunk 2]
```diff
@@ -54,8 +55,18 @@ const TextEdit: FC<CombinedProps> = (props) => {
     if (value !== propValue) {
       setValue(propValue)
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [propValue])
 
+  useEffect(() => {
```
Comment (JonaszJestem) [inline src/frontend/components/property-type/default-type/edit.tsx:61]: trigger record change when typing in the field with error

**Review by SimonB407 — APPROVED** [review]
Two questions. can you describe, why in so many files you disabled `eslint-disable react-hooks/exhaustive-deps`? And also are you sure that `admin-bro/design-system beta 24` is safe? Because I remember we had troubles with some version of beta design system.

**Review by Co0sh — APPROVED** [review]

**Comment by dziraf** [comment]
Closing because I'm not sure why `eslint-disable react-hooks/exhaustive-deps` needed to be disabled and this PR is already old

**Pull request closed without merging**